packages/datastore/package.json

```diff
@@ -56,6 +56,7 @@
 		"amazon-cognito-identity-js": "5.2.10",
 		"idb": "5.0.6",
 		"immer": "9.0.6",
+		"semver": "7.3.7",
```

Comment:
Added a new dep here, but additional size to bundle with partial import `import semverSatisfies from 'semver/functions/satisfies';`. We could roll our own solution but would take a little extra time.

Reply:
Yeah, this is pretty heavy for a version check that we control on both ends. If this blows bundle size, let's start with a more rudimentary check.

For our purposes, we can start by asserting that the "supported version" string is _simple_, in the form of `^N.N.N`, and then that each part of the "version to check" is `>=` the expected part. I don't think we'll need anything more sophisticated than that for quite awhile. And, we can safeguard against bad assumptions by moving the expected version to a constant and writing a test around it to ensure it's a "simple" semver matcher.